src/test/java/picard/fingerprint/CrosscheckFingerprintsTest.java

```diff
@@ -39,9 +40,21 @@
     private final int NA12892_r1_RGs = 25;
     private final int NA12892_r2_RGs = 26;
 
+    private static File  NA12891_1_vcf;
+    private static File  NA12891_2_vcf;
+    private static File  NA12892_1_vcf;
+    private static File  NA12892_2_vcf;
+    private static File  NA12891_named_NA12892_vcf;
+    private static File  NA12891_g_vcf;
+    private static File  NA12892_g_vcf;
+    private static File  NA12892_and_NA123891_vcf;
+    private static File  NA12892_and_NA123891_part1_vcf;
+    private static File  NA12892_and_NA123891_part2_vcf;
+    private static File  NA12892_and_NA123891_part3_vcf;
+
     private static final Map<CrosscheckMetric.DataType, List<String>> lookupMap = new HashMap<>(4);
     
-    @BeforeTest
+    @BeforeSuite
```

Comment:
You probably want to use before class instead so this doesn't run if you don't run this test class.

Reply:
done.